packages/client/package.json

```diff
@@ -10,45 +10,35 @@
     "start": "cross-env BROWSER=none react-scripts start",
     "prebuild": "cd ../common && yarn build",
     "build": "react-scripts build",
+    "lint": "eslint src/ --ext .ts,.tsx",
+    "prepare": "cd ../.. && yarn prepare",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "codegen": "graphql-codegen --config codegen.yml && scripts/patch-generated-graphql",
-    "codegen:watch": "watch 'yarn codegen' ../main/schema src/documents",
-    "postinstall": "cd ../.. && yarn postinstall"
+    "codegen:watch": "watch 'yarn codegen' ../main/schema src/documents"
   },
   "eslintConfig": {
     "extends": "react-app"
   },
   "browserslist": [
-    "electron >= 4"
+    "electron >= 6"
   ],
   "dependencies": {
     "@material-ui/core": "^4.0.0",
     "@material-ui/icons": "^4.0.0",
     "@reach/router": "^1.2.1",
-    "@types/autosuggest-highlight": "^3.1.0",
-    "@types/graphql": "^14.2.2",
-    "@types/is-url": "^1.2.28",
-    "@types/marked": "^0.6.5",
-    "@types/node": "11.11.3",
-    "@types/object.pick": "^1.3.0",
-    "@types/reach__router": "^1.2.4",
-    "@types/react": "^16.8.22",
-    "@types/react-autosuggest": "^9.3.8",
-    "@types/react-dom": "^16.8.4",
-    "@types/string-hash": "^1.1.1",
```

Comment:
I moved `@types/*` packages to dev dependencies because they are only useful during type checking, and do not need to be packaged with the final build of the app.